server/market/market.go

```diff
@@ -1307,6 +1307,10 @@ func (m *Market) Unbook(lo *order.LimitOrder) bool {
 	_, removed := m.book.Remove(lo.ID())
 	m.bookMtx.Unlock()
 
+	// TODO: Should not unlock coins if there are active/unrevoked
+	// matches for this order and no swaps have been sent.
+	// If any swap has been sent, the funding coins would have been
+	// spent and it would be theoretically safe to unlock the coins.
```

Comment:
I don't think this comment is right because the swapper has it's own coin locker, and the "is locked" check when processing new orders requires **both** the market's and swapper's lockers to have unlocked to consider coins unlocked (conversely, if either the swapper or market has the coin locked, it is considered locked).

Reply:
Referring to https://github.com/decred/dcrdex/blob/7671c15474684ccb4e712be0de4939e66de7d3e0/server/coinlock/coinlocker.go#L53-L61

Reply:
Ah, gotcha. Will remove the comment.